doc/source/user/basics.types.rst

```diff
@@ -342,6 +342,35 @@ range of possible values.
     >>> np.power(100, 100, dtype=np.float64)
     1e+200
 
+Floating point precision
+========================
+
+Many functions in NumPy, especially those in `numpy.linalg`, involve floating-point
+arithmetic, which can introduce small inaccuracies due to the way computers 
+represent decimal numbers. For instance, when computing the determinant of the 
+following matrix:
+
+    >>> import numpy as np
```

Comment:
You could maybe drive this home even more with a simpler example. My favorite is
```python
>>> 0.3 - 0.2 - 0.1
-2.7755575615628914e-17
```
Though this uses Python float arithmetic rather than NumPy.